requirements.yml

```diff
@@ -1,5 +1,11 @@
 ---
 
+collections:
+- name: community.general
+  version: latest
```

Comment:
CI error:
https://github.com/collectionspace/cspace-installer/actions/runs/4681444552/jobs/8294467153

Appears related to:
https://bugs.launchpad.net/tripleo/+bug/1982552

Replicated locally:
```
Using /home/mark-cooper/Projects/collectionspace/cspace-installer/ansible.cfg as config file
[DEPRECATION WARNING]: [defaults]callback_whitelist option, normalizing names to new standard, use callbacks_enabled instead. This feature will be removed from ansible-core in version 2.15. Deprecation 
warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
Starting galaxy collection install process
Process install dependency map
ERROR! Non integer values in LooseVersion ('latest')
```

Using specific version (`6.5.0`) succeeds.